src/lib/radio/radio.ts

```diff
@@ -516,4 +498,20 @@ export class MdRadioButton implements OnInit, AfterViewInit, OnDestroy {
     }
   }
 
+  /** Function is called whenever the focus changes for the input element. */
+  private _onInputFocusChange(focusOrigin: FocusOrigin) {
```

Comment:
The JSdoc reads to me like it's only when it receives focus. Either way this really minor, I'll delete the note.